src/bosh-director/db/migrations/migration_digests.json

```diff
@@ -145,5 +145,6 @@
   "20180411165035_migrate_update_strategy_to_vm_strategy": "116a03febc41f331316bb0d14385093ed941bc68",
   "20180427184448_add_historical_metadata_to_orphaned_vms": "405597203ab69f062481a9896a667617bc3207f7",
   "20180515204145_add_vm_metadata_spec": "3a094f32feef551247aeb82432cb551d4804ac16",
-  "20180613190204_links_api_longtext_support": "555ab4f70cc81a7a5889429f8ddadb08dcd432f0"
-}
\ No newline at end of file
+  "20180613190204_links_api_longtext_support": "555ab4f70cc81a7a5889429f8ddadb08dcd432f0",
+  "20180327200711_create_network_table": "167d33bbd968ad671f2c1f7bac5d5046e77edd4f"
```

Comment:
Note to merger: rename this migration to be the most recent one (use current date).